spec/pom.xml

```diff
@@ -190,10 +190,10 @@
 				</executions>
 				<configuration>
 					<sourceDocumentName>cdi-spec.asciidoc</sourceDocumentName>
-					<sourceHighlighter>coderay</sourceHighlighter>
 					<attributes>
 						<license>${license-file}</license>
 						<revremark>${revremark}</revremark>
+						<source-highlighter>coderay</source-highlighter>
```

Comment:
IIRC, these code highlighters only work if you have them locally installed when building the doc.
There is one exception to it - `highlight.js`.

So I have two questions:
* Would it be worth having `highlight.js` for HTML doc build and something else for PDF? So that local builds for anyone would have highlighting in HTML at least.
* Are we sure our Jenkins machines have this ruby gem installed? (for releases)

Reply:
I have tested this PR using the https://ci.eclipse.org/cdi/job/CDISpec job. There are no errors building the specs html/pdf and the code fragments look good in both. See https://ci.eclipse.org/cdi/job/CDISpec/8/ build artifacts. You have to download the html version for it to display correctly.